advanced/postgres_lambda_strategy/main.tf

```diff
@@ -56,10 +56,12 @@ resource "sym_integration" "lambda_context" {
 
 # A target AWS Lambda that will be invoked on escalate and de-escalate.
 # The `name` will be used in the lambda to decide which resource to manage access to
-resource "sym_target" "readonly" {
+resource "sym_target" "postgres_roles" {
+  for_each = { for target in var.postgres_roles : target["name"] => target["label"] }
```

Comment:
I'm not totally convinced this simplifies the docs enough to be worth it? I don't feel super strongly, but my initial thoughts are:
- for_each has caused trouble with changes / tf state / sym references in the past
- the docs are just a `tfvars` definition vs a `sym_target` definition, both are pretty simple
- while the `tfvars` seem simpler on the surface, readers actually don't know what it's doing, which isn't necessarily simpler in practice (this is part of why we ended up removing tfvars from other examples)

Reply:
I hear you, i just think in this case it is nice for the docs example to only require you to make changes in once place (tfvars). I think the issues with for_each were due to funky key naming and we've Moved On.